tensorflow-framework/src/main/java/org/tensorflow/framework/activations/ReLU.java

```diff
@@ -93,9 +95,66 @@ public ReLU(float alpha, float maxValue, float threshold) {
     this.threshold = threshold;
   }
 
-  /** {@inheritDoc} */
-  @Override
-  public Operand<T> call(Ops tf, Operand<T> input) {
+  /**
+   * Creates a ReLU activation from a config map.
+   *
+   * @param config the configuration map,
+   *     <ul>
+   *       <li>if the map contains an entry for {@code alpha} that value is used, otherwise {@link
+   *           #ALPHA_DEFAULT} is used.
+   *       <li>if the map contains an entry for {@code max_value} that value is used, otherwise
+   *           {@link #MAX_VALUE_DEFAULT} is used.
+   *       <li>if the map contains an entry for {@code threshold} that value is used, otherwise
+   *           {@link #THRESHOLD_DEFAULT} is used.
+   *     </ul>
+   */
+  public ReLU(Map<String, Object> config) {
+    this(
+        ((Number) config.getOrDefault("alpha", ALPHA_DEFAULT)).floatValue(),
```

Comment:
Why cast to `Number` and get floatValue rather than casting to `Float`?

Reply:
The number is converted to Double in the (GSON)  json parser.